README.md

```diff
@@ -1,2 +1,41 @@
 # -Hello-Microverse-project
-"Hello Microverse" project first project Day2
+
+
+Hello-Microverse-project
+First Project excuted on Day2 of the program
+
+Built With
+HTML & CSS
+using Vscode & Github
+
+Getting Started
+To get a local copy up and running follow these simple example steps.
+
+Setup
+Install
+Usage
+Deployment
+
+Authors
+👤 Paul Malongo
+
+GitHub: @canon20paul
+Twitter: @paulmalongo
+LinkedIn: Paulmalongo
+
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support
+Give a ⭐️ if you like this project!
+
+Acknowledgments
+Sincerely appreciate the Microverse Team for igniting this journey
+etc
+
+📝 License
+
+ License
+This project is MIT licensed.
```

Comment:
- [ ] As stated by the previous reviewer, you should use a microverse template and the same markdown syntax, you need to copy syntax also not the text, so in [this link](https://raw.githubusercontent.com/microverseinc/readme-template/master/README.md) you can copy text with syntax and customize it for your repo to make it a professional one.